app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::EmbeddedAnsible::CrudCommon
   extend ActiveSupport::Concern
 
+  def self.included(model)
+    model.include SupportsFeatureMixin
```

Comment:
Typically this is done in each base model, weird to have it here since this will be mixed-in to child classes

Reply:
@agrare We have been doing a lot of work to remove the voodoo magic in terms of all these `supports` calls.

You thinking we should add it to
- app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb
- app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

I also noticed that in tower, it gets mixed into `configuration_script`. Do we think these 2 should be different?